src/holofuel/components/Header/Header.js

```diff
@@ -25,9 +26,9 @@ export function Header ({ title, agent, agentLoading, avatarUrl, history: { push
         </div>}
       </div>
       <div>
-        <CopyAgentId agent={agent} isMe>
-          <HashAvatar avatarUrl={avatarUrl} seed={agent.id} size={32} dataTestId='hash-icon' />
-        </CopyAgentId>
+        <Link to={PROFILE_PATH}>
+          <HashAvatar seed={agent.id} size={32} dataTestId='hash-icon' />
+        </Link>
```

Comment:
I think we want to keep the copy functionality. Not sure. We went back and forth on this one and not sure where we landed. But I missed it when testing

Reply:
This is definitely a UX decision.  I'm fine with either implementation for now.